src/sfizz/modulations/sources/Controller.cpp

```diff
@@ -58,6 +75,7 @@ void ControllerSource::init(const ModKey& sourceKey, NumericId<Voice> voiceId, u
     if (p.smooth > 0) {
         Smoother s;
         s.setSmoothing(p.smooth, impl_->sampleRate_);
+        s.reset(impl_->getLastTransformedValue(p.cc, p.curve));
```

Comment:
I added this, I think this function can be call with a "non-zero" cc state, in which case the smoother should start at the current cc value I think.

Reply:
Imo, the smoother would rather keep the last target value given to it, and have something along the lines of `Smoother::resetToTarget`. The linear one already has this.
What do you think?

Reply:
Sure, why not ! I'll add it.

Reply:
OK actually not a good idea, because usually when the playhead moves the host sends new CC value. You don't want to reset to the previous target, you want to reset to the new CC state so you need to query it fresh.

I did add the `resetToTarget` methods though, might keep them.